opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -260,15 +260,22 @@ def test_invalid_instrumentation_info(self):
         self.assertEqual(
             tracer1.instrumentation_info, tracer2.instrumentation_info
         )
+
         self.assertIsInstance(
             tracer1.instrumentation_info, InstrumentationInfo
         )
         span1 = tracer1.start_span("foo")
         self.assertTrue(span1.is_recording())
         self.assertEqual(tracer1.instrumentation_info.version, "")
-        self.assertEqual(
-            tracer1.instrumentation_info.name, "ERROR:MISSING MODULE NAME"
+        self.assertEqual(tracer1.instrumentation_info.name, "")
+
```

Comment:
Remove this empty line?

Reply:
Good question; I think this empty line helps the readability in context, though.